wiki/Gameplay/Grade/en.md

```diff
@@ -17,39 +26,35 @@ needs_cleanup: true
 | C | Over 60% 300s |
 | D | Anything else |
 
-A silver S or SS can be obtained by using the *[Hidden](/wiki/Game_modifier/Hidden)* and/or *[Flashlight](/wiki/Game_modifier/Flashlight)* mod.
-
 ## osu!taiko
 
 | Grade | Condition |
 | :-: | :-- |
-| SS | 100% Accuracy. |
-| S | Over 95% Accuracy (90+% GREATs, or less than 1 GOOD for every 10 notes). |
-| A | Over 90% Accuracy (80+% GREATs, or less than 1 GOOD for every 5 notes). |
-| B | Over 80% Accuracy (70+% GREATs, or less than 1 GOOD for every 3.33 notes). |
+| SS | 100% accuracy |
+| S | Over 90% GREATs and no misses |
+| A | Over 80% GREATs and no misses **OR** over 90% GREATs |
+| B | Over 70% GREATs and no misses **OR** over 80% GREATs |
+| C | Over 60% GREATs |
+| D | Any other passing score |
```

Comment:
@bdach What are the clusivities of these ranges (`>` or `>=`)? Same for osu!, taiko and mania, and I presume catch is already almost unambiguous

Reply:
All ranges are exclusive, with the single exception of the 50s clause for S rank in osu! (as in you can have exactly 1% of all judgements be 50s in osu! and still score an S rank if the accuracy is otherwise good enough, but not more).